CONTRIBUTING.md

```diff
@@ -42,6 +42,8 @@ are the same under GNU/Linux and Mac OSX:
 
 ##### C files
 
+Gregorio uses [ISO C99 Standard](http://en.wikipedia.org/wiki/C99), and targets the [gcc](https://gcc.gnu.org/) and [clang](http://clang.llvm.org/) compilers only. Patches for other compilers can be accepted if they are clean, small and non-intrusive.
```

Comment:
I think it should be "Gregorio uses the [ISO C99 Standard]..."  Also see my comment about GCC / Clang extensions in the pull request.